RadixWallet/Features/DerivePublicKeys/DerivePublicKeys.swift

```diff
@@ -89,6 +89,8 @@ public struct DerivePublicKeys: Sendable, FeatureReducer {
 						.device
 					case let .ledger(ledger):
 						.ledger(ledger)
+					default:
+						fatalError("DISCREPANCY: Found non .device | .ledger factor source. A real world user cannot possible have this.")
```

Comment:
This is safe for our users, it is not possible for our users to create any other factor source than `DeviceFactorSource` or `LedgerFactorSource`.